src/test/shared/sam/cli/samCliCommand.test.ts

```diff
@@ -0,0 +1,32 @@
+/*!
+ * Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+
+import * as assert from 'assert'
+import { SamCliInfoResponse, SamInfoCliCommand } from '../../../../shared/sam/cli/samCliCommand'
+
+describe('SamInfoCliCommand', async () => {
+
+    it('converts sam info response to SamCliInfoResponse', async () => {
```

Comment:
Minor: Suggest to add test case for `'{"version": "1.2.3"} then some unrelated text'`